mets.md

```diff
@@ -160,6 +199,83 @@ encodings of the same page.
 </mets:structMap>
 ```
 
+## OCR-D structMap
+
+A METS can contain different `structMap`. These are defined by the `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`>) described in more detail.	
+A document structure determined by OCR-D is described by `TYPE="OCR-D-LOGICAL"`.
```

Comment:
What exactly does this mean? There's also the mandatory `PHYSICAL` layout. So maybe this should read: 
```suggestion
The _logical_ document structure detected by MP MUST be described by `TYPE="OCR-D-LOGICAL"`.
```

Reply:
```suggestion
The _logical_ document structure detected by OCR-D software MUST be described by `TYPE="OCR-D-LOGICAL"`.
```

Reply:
I'd be okay with either, but AFAICS the term _MP_ (for module project) has been used more often so far.

Maybe we should revisit this, anyway – the spec should just call them "implementors" or "implementing software" or "OCR-D software"